src/cn-flex-form.service.js

```diff
@@ -1783,7 +1793,18 @@ function CNFlexFormService(
     const service = this;
     const selectFieldKey = service.getKey(selectField.key);
 
-    _.each(selectDisplay.items, item => {
+    let selectDisplayItems = [];
+    // Needed for batchform to check recursively
+    for (let item of selectDisplay.items) {
+      if (!item.items) {
+        selectDisplayItems.push(item);
+      } else if (item.items) {
+        selectDisplayItems.push(...item.items);
+      }
+    }
```

Comment:
Same disclaimer, but this looks like it could be:

```js
let selectDisplayItems =
  _(selectDisplay.items)
  .map(item => item.items || [item])
  .flatten()
  .value();
```

Reply:
This, however, is definitely better than what I wrote originally! 🙌